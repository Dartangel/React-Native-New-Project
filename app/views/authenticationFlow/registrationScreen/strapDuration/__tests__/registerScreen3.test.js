import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import StrapDuration from '../strapDuration';

describe('StrapDuration', () => {
    test('StrapDuration snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<StrapDuration />);
        expect(snapshot).toMatchSnapshot();
    });
});