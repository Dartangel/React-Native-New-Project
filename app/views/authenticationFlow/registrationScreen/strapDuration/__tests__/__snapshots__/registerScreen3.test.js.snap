// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StrapDuration StrapDuration snapshot 1`] = `
<ImageBackground
  source={
    Object {
      "testUri": "../../../app/src/img/splashScreen1.jpeg",
    }
  }
  style={
    Object {
      "height": "100%",
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Header
    header="Сколько Вы можете 
 простоять в планке?"
    subHeader="Ваши данные помогут нам сделать ваши 
 					 тренировки эффективнее"
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
      }
    }
  >
    <Picker
      itemSpace={50}
      itemStyle={null}
      onValueChange={[Function]}
      pickerData={
        Array [
          "Более 120c",
          "0-30c",
          "30-60c",
          "60-120c",
        ]
      }
      selectedValue="0-30"
      style={
        Object {
          "alignContent": "center",
          "backgroundColor": "white",
          "height": "65%",
          "paddingTop": 20,
          "width": "100%",
        }
      }
      textColor="#333"
      textSize={26}
    />
    <NextButton
      onPress={[Function]}
      title="Cледующие"
    />
  </View>
</ImageBackground>
`;
